cards/src/main/resources/cards/custom/group6/witch_doctor/secrets/secret_subject_to_sacrifice.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Subject to Sacrifice",
+  "baseManaCost": 2,
+  "type": "SPELL",
+  "heroClass": "ROSE",
+  "rarity": "EPIC",
+  "description": "Secret: After your opponent plays a minion, all your minions attack it.",
+  "secret": {
+    "class": "MinionPlayedTrigger",
+    "fireCondition": {
+      "class": "MinionCountCondition",
+      "value": 0,
+      "operation": "GREATER"
+    },
+    "targetPlayer": "OPPONENT"
+  },
+  "spell": {
+    "class": "FightSpell",
+    "target": "FRIENDLY_MINIONS",
```

Comment:
i believe it should be `"target": "EVENT_TARGET"`, `"secondaryTarget": "FRIENDLY_MINIONS"`. Write this one a test because it's a little wonky.